soc/arm/cypress/Kconfig

```diff
@@ -0,0 +1,33 @@
+#
+# Copyright (c) 2018, Cypress
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+choice
+prompt "Cypress PSoC6 MCU Selection"
+depends on SOC_SERIES_PSOC62
+
+config SOC_PSOC6_M0
+	bool "SOC_PSOC6_M0"
+	select CPU_CORTEX_M0PLUS
+	select CPU_CORTEX_M_HAS_VTOR
+
+config SOC_PSOC6_M4
+	bool "SOC_PSOC6_M4"
+	select CPU_CORTEX_M4
```

Comment:
Should we select CPU_HAS_FPU

Reply:
When I add "select CPU_HAS_FPU" after "select CPU_CORTEX_M4" it turns on CONFIG_FLOAT and the problem with math.h include appears.
What is better to do here?

Reply:
Lets leave it alone for now, and I'll take a look after we get this merged